heft-plugins/heft-serverless-stack-plugin/src/ServerlessStackPlugin.ts

```diff
@@ -35,44 +36,39 @@ export default class ServerlessStackPlugin implements IHeftTaskPlugin {
 
     // Once https://github.com/serverless-stack/serverless-stack/issues/1537 is fixed, we may be
     // eliminate the need for this parameter.
-    const sstParameter: CommandLineFlagParameter = taskSession.getFlagParameter('--sst');
-    const sstStageParameter: CommandLineStringParameter = taskSession.getStringParameter('--sst-stage');
+    const sstParameter: CommandLineFlagParameter = taskSession.parameters.getFlagParameter('--sst');
+    const sstStageParameter: CommandLineStringParameter =
+      taskSession.parameters.getStringParameter('--sst-stage');
 
     // Only tap if the --sst flag is set.
     if (sstParameter.value) {
-      const configureWebpackTapOptions: { name: string; stage: number } = {
-        name: PLUGIN_NAME,
-        stage: Number.MAX_SAFE_INTEGER
```

Comment:
Do we not need to run this last anymore?

Reply:
Correct, this is an `AsyncSeriesBailHook`. If any plugin taps this and returns a value (of type `IWebpackConfiguration | false`), that becomes the result of calling the tap. So no, it doesn't need to run last anymore. However, there is a good argument for it needing to run _first_, since it is disabling Webpack entirely. Can change if you think it's justified.